Ember/README.md

```diff
@@ -14,6 +14,83 @@
 ## Naming Conventions
 We follow the canonical naming conventions established by the Ember Core Team. The full Ember naming conventions documentation can be found [here](http://emberjs.com/guides/concepts/naming-conventions/).
 
+### Route naming conventions:
+On both the server-side and client-side of Mercury application, route handlers should be camelCased and route paths should be dasherized.
+No trailling slash on the end of the path.
+
+Server side:
+```javascript
+	// good
+	{
+		method: 'POST',
+		path: '/article-preview',
+		handler: articlePreview
+	}
+
+	// bad
+	{
+		method: 'POST',
+		path: '/articlePreview',
+		handler: articlePreview
+	}
+
+	// bad
+	{
+		method: 'POST',
+		path: '/articlePreview',
+		handler: article-preview
+	}
+
+	// bad
+	{
+		method: 'POST',
+		path: '/article-preview',
+		handler: articlePreview/
```

Comment:
While this is also bad for other reasons ;) this trailing slash example should be on the path rather than the handler, right?